_includes/tutorials/sliding-windows/kstreams/markup/dev/generate-data.adoc.bak

```diff
@@ -1,44 +0,0 @@
-////
-   Example content file for how to include a console producer(s) in the tutorial.
-   Usually you'll include a line referencing the script to run the console producer and also include some content
-   describing how to input data as shown below.
-
-   Again modify this file as you need for your tutorial, as this is just sample content.  You also may have more than one
-   console producer to run depending on how you structure your tutorial
-
-////
-
-Before you start your Kafka Streams application, we need to provide data for it. Fortunately this is as simple as using a HTTP `PUT` request, as you're going to use the `DatagenConnector`.
-
-
-Now create the following Avro schema file `datagen-temperature-reading.avsc` in the current working directory (`sliding-windows`) for the tutorial:
-
-+++++
-<pre class="snippet"><code class="avro">{% include_raw tutorials/sliding-windows/kstreams/code/datagen-temperature-reading.avsc %}</code></pre>
-+++++
-
-This schema file is pretty much idential to the one you created earlier.
-
-The only difference is this schema contains instructions for data generation.  The https://github.com/confluentinc/kafka-connect-datagen[kafka-connect-datagen] connector uses the https://github.com/confluentinc/avro-random-generator[Avro Random Generator] to generate data.
-
-
-Open an new terminal window and run this command to start the data generator:
-+++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/sliding-windows/kstreams/code/tutorial-steps/dev/create-connector.sh %}</code></pre>
-+++++
-
-
-You should see something like this on the console indicating the datagen connector sucessfuly started
-
-----
-HTTP/1.1 201 Created
-Date: Tue, 26 Jan 2021 21:40:33 GMT
-Location: http://localhost:8083/connectors/datagen_local_01
-Content-Type: application/json
-Content-Length: 413
-Server: Jetty(9.4.24.v20191120)
-
-{"name":"datagen_local_01","config":{"connector.class":"io.confluent.kafka.connect.datagen.DatagenConnector","key.converter":"org.apache.kafka.connect.storage.StringConverter","kafka.topic":"temp-readings","schema.filename":"/schemas/datagen-temperature-reading.avsc","schema.keyfield":"device_id","max.interval":"300","iterations":"10000000","tasks.max":"1","name":"datagen_local_01"},"tasks":[],"type":"source"}
-----
-
-
```

Comment:
This file appears to have been committed by accident before, so deleting this as cleanup on the side